spec/factories/cargos.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :cargo do
+    description { FFaker::Lorem.paragraph }
+    title { FFaker::Lorem.word }
+    reference_number { rand(0..999999) }
+    value { FFaker.numerify("#.##") }
```

Comment:
if you want to just use rand for this as its a decimal you can do:
`rand(0.00..10_000.00).round(2)` if you put decimals with a spread so `1.00..10.00` this will then result in a value like: `1.11111111`